static/js/redux/ui/course_list_row.jsx

```diff
@@ -165,6 +165,8 @@ CourseListRow.propTypes = {
   courseToClassmates: PropTypes.shape({ '*': SemesterlyPropTypes.classmates }).isRequired,
   fetchCourseInfo: PropTypes.func.isRequired,
   timetableName: PropTypes.string.isRequired,
+  // TODO: fix proptype validation of selected_advisee
+  selected_advisee: PropTypes.PropTypes.objectOf(PropTypes.string),
```

Comment:
its not an object of strings!

Reply:
I know, that's why I put the TODO above it lol